sing in.html

```diff
@@ -0,0 +1,43 @@
+<html>

```

Comment:
### \<\!DOCTYPE html\> is missing
This tag provide information and rules to browsers to render page content.
Forget it and you will encounter UNPREDICTED rendering.

[More about this tag](https://graybox.co/knowledge/blog/the-importance-of-the-doctype#:~:text=Per%20HTML%20and%20XHTML%20standards,correctly%20or%20validate%20without%20them.)